extensions/amp-analytics/amp-analytics.md

```diff
@@ -27,70 +27,72 @@ limitations under the License.
+
+In the following example, analytics data is sent to Nielsen, a third-party analytics provider. Details for configuring analytics data for Nielsen can be found in the [Nielsen](https://engineeringportal.nielsen.com/docs/DCR_Static_Google_AMP_Cloud_API) documentation.
 
 ```html
-<amp-analytics>
-<script type="application/json">
-{
-  "requests": {
-    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
-    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
-  },
-  "vars": {
-    "account": "ABC123"
-  },
-  "triggers": {
-    "trackPageview": {
-      "on": "visible",
-      "request": "pageview"
-    },
-    "trackAnchorClicks": {
-      "on": "click",
-      "selector": "a",
-      "request": "event",
+<amp-analytics type="nielsen">
+    <script type="application/json">
+    {
       "vars": {
-        "eventId": "42",
-        "eventLabel": "clicked on a link"
+        "apid": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
+        "apv": "1.0",
+        "apn": "My AMP Website",
+        "section": "Entertainment",
+        "segA": "Music",
+        "segB": "News",
+        "segC": "Google AMP"
       }
     }
-  }
-}
-</script>
+    </script>
 </amp-analytics>
 ```
 
-## Attributes
+### Sending data in-house
 
-**type**
+If you have your own in-house solution for measuring user engagement, the only thing you will need to integrate AMP analytics with that solution is a URL. This is where you will send the data. You can also send data to various URLs. For example, you can send page view data to one URL, and social engagement data to another URL.
 
-Specifies the type of vendor.  For details, see the [Analytics vendors](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html) page.
+To send data in-house:
 
-**config**
+1.  Determine what data you want to capture and track, and [specify those details in the configuration data](#specifying-configuration-data).
+2.  In the [`requests`](#requests) configuration object, specify the url(s) of where you want to send the tracking data to.
 
-This is an optional attribute that can be used to load a configuration from a specified remote URL. The URL specified should use the HTTPS scheme. See also the `data-include-credentials` attribute below. The URL may include [AMP URL vars](../../spec/amp-var-substitutions.md). The response must follow the [AMP CORS security guidelines](../../spec/amp-cors-requests.md).
+*Example: Sending data to a URL*
 
-Example:
+In the following example, everytime a page is visible, the trigger event fires, and sends the pageview data to a defined URL along with a random ID:
 
 ```html
-<amp-analytics config="https://example.com/analytics.config.json"></amp-analytics>
-<amp-analytics type="googleanalytics" id="analytics1">
+<amp-analytics>
 <script type="application/json">
 {
-  "vars": {
-    "account": "UA-12345-Y"
+  "requests": {
+    "pageview": "https://foo.com/pixel?RANDOM",
```

Comment:
I just meant that the docs don't make it clear that other things besides page views can be tracked at all.